yarn-project/noir-compiler/src/__snapshots__/index.test.ts.snap

```diff
@@ -38,6 +39,7 @@ exports[`should compile the test contract 1`] = `
       {
         "bytecode": "00000000020000000000000001000000010000000100000000000000000100000030644e72e131a029b85045b68181585d2833e84879b9709143e1f593f000000001000000000000000000000000000000000000000000000000000000000000000000002a",
         "functionType": "open",
+        "isInternal": undefined,
```

Comment:
was the undefined case handled for ealier?

Reply:
Seems like it is using an old noir wasm (0.5.1) and don't seem like there is any version in the last two months (https://www.npmjs.com/package/@noir-lang/noir_wasm) 😅 
Without those it will keep reading undefined for the `isInternal`.

@sirasistant @kobyhallx you have input here, is there a reason the noir_wasm are so stale?

Reply:
Seems like the gh action is missing auth  https://github.com/noir-lang/noir_wasm/actions/runs/5662604210

Reply:
Im assuming this will get fixed over time, and as this folder is not under active development, this is fine for now